File: client/src/components/Lessons/Lesson.js
import './Lessons.css'

const Lesson = ({ objective, repertoire, assignment, earned_grade, date}) => {
    const formattedDate = date.strftime()

    console.log(formattedDate)
    
    return (
        <div className='single-lesson-container'>
            <h3>Lesson Date: {date}</h3>
        </div>
    )
}

export default Lesson

File: app/models/lesson.rb
class Lesson < ApplicationRecord
  belongs_to :student, foreign_key: "user_id", class_name: "User"
  has_many :comments, as: :commentable

  validates :date, presence: true
  validates :objective, presence: true
  validates :repertoire, presence: true
  validates :assignment, presence: true
  
end
